manifests/a/Alex313031/Codium/1.86.2.24053/Alex313031.Codium.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Alex313031.Codium
+PackageVersion: 1.86.2.24053
+InstallerType: inno
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Alex313031/codium/releases/download/1.86.2.24053/Codium_Setup_x64_1.86.2.24053.exe
+  InstallerSha256: B066D6A6B1B7734D49C618113728ABEB344A90537A3BE526DC1CC28DB50C9661

```

Comment:
```suggestion
  InstallerSha256: B066D6A6B1B7734D49C618113728ABEB344A90537A3BE526DC1CC28DB50C9661
  Scope: machine
- Architecture: x64
  InstallerUrl: https://github.com/Alex313031/codium/releases/download/1.86.2.24053/Codium_User_Setup_x64_1.86.2.24053.exe
  InstallerSha256: c68aeb1364efec96ea545d630a309fddc1176dade53ae0820bff0c5fc0737018
  Scope: user
```

Reply:
For many users using multiple accounts, it will be helpful to install it machine-wide.